xapi-nbd.opam

```diff
@@ -1,11 +1,13 @@
-opam-version: "1.2"
+opam-version: "2.0"
 maintainer: "xen-api@lists.xen.org"
 authors: ["dave.scott@citrix.com"]
 homepage: "https://github.com/xapi-project/xapi-nbd"
 bug-reports: "https://github.com/xapi-project/xapi-nbd/issues"
-dev-repo: "https://github.com/xapi-project/xapi-nbd.git"
-build: [[ make "release" ]]
-build-test: [[ "dune" "runtest" ]]
+dev-repo: "git+https://github.com/xapi-project/xapi-nbd.git"
+build: [
+  [ make "release" ]
+  [ "dune" "runtest" ] {with-test}
```

Comment:
This can be done using
```
run-test: ["dune" "runtest" "-p" name "-j" jobs]
```
See https://dune.readthedocs.io/en/latest/usage.html#invocation-from-opam for reference.